main/src/ca/uwaterloo/flix/tools/CompilerPerf.scala

```diff
@@ -267,7 +268,7 @@ object CompilerPerf {
       ("lines" -> lines) ~
       ("results" -> baseline.phases.zip(baselineWithPar.phases).map {
         case ((phase, times1), (_, times2)) =>
-          ("phase" -> phase) ~ ("speedup" -> combine(times1) / combine(times2))
+          ("phase" -> phase) ~ ("speedup" -> combine(times1.zip(times2).map(p => p._1 / p._2)))
```

Comment:
Imagine if I could do `map(/)` or `map(Div.div)`-- Well in Flix I will soon be able!

Reply:
Hell, even today I can do `map(match (x, y) -> x / y`.